pkg/piecestore/rpc/client/api/api.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package api
+
+import (
+	"errors"
+	"io"
+	"log"
+
+	"github.com/zeebo/errs"
+
+	"golang.org/x/net/context"
+
+	pb "storj.io/storj/protos/piecestore"
+)
+
+var serverError = errs.Class("serverError")
+
+// PieceMeta -- Struct containing Piece information from PieceMetaRequest
+type PieceMeta struct {
+	Hash       string
+	Size       int64
+	Expiration int64
+}
+
+// PieceMetaRequest -- Request info about a piece by Hash
+func PieceMetaRequest(ctx context.Context, c pb.PieceStoreRoutesClient, hash string) (*PieceMeta, error) {
```

Comment:
could this be defined on some kind of Client wrapper that wraps a pb.PieceStoreRoutesClient so you don't have to keep passing in c to all of these client calls? a bit more object oriented